osbenchmark/builder/models/flavors.py

```diff
@@ -0,0 +1,6 @@
+from enum import Enum
+
+
+class Flavor(str, Enum):
```

Comment:
Thoughts on calling this `ClusterFlavor` instead?

Reply:
That's more clear, will change